lib/success.js

```diff
@@ -15,7 +15,10 @@ module.exports = async (pluginConfig, context) => {
 		options,
 		env: { SEMANTIC_RELEASE_PACKAGE, npm_package_name }
 	} = context
-	const { slackWebhook = process.env.SLACK_WEBHOOK } = pluginConfig
+	const {
+		slackWebhook = process.env.SLACK_WEBHOOK,
+		maxLength = MAX_LENGTH
```

Comment:
Let's change `maxLength` to something with 'unsafe' in it.This change also has to happen in README.md

Reply:
Done